src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -2,10 +2,7 @@
 
 import java.time.LocalDate;
 import java.time.LocalTime;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Optional;
-import java.util.Set;
+import java.util.*;
```

Comment:
I think inteiilj auto made the star again, i'll revert the stars back in my commits